rhine/src/FRP/Rhine/Clock/Periodic.hs

```diff
@@ -45,7 +45,7 @@ instance (Monad m, NonemptyNatList v)
   type Time (Periodic v) = Integer
   type Tag  (Periodic v) = ()
   initClock cl = return
-    ( cycleS (theList cl) >>> withSideEffect wait >>> sumS &&& arr (const ())
+    ( cycleS (theList cl) >>> withSideEffect wait >>> (accumulateWith (+) 0) &&& arr (const ())
```

Comment:
`sumS` still exists in `dunai` afaikt, you have to `import Data.MonadicStreamFunction.Util`.

Reply:
True, but the new sumS implementation requires instances of The VectorSpace. Integer isn't a VectorSpace.

Reply:
Fair enough.